docs/intentReaderHandlers/ClearEventListeners.md

```diff
@@ -0,0 +1,61 @@
+# ClearEventListeners
+
+## Overview
+
+ClearEventListeners handler is invoked when the task specified in the intent has the value "clearAllListeners". This handler is used to deregister all the events that are registered already. 
+
+It performs the following actions :
+1. Deregister all the events.
+2. Save the response/error.
+3. Format the result and send the response back to IntentReader.
+
+## Usage
+* This handler is invoked to clear all the event listeners at a time.
+
+```json
+    {
+        "action": "search",
+        "data": {
+            "query": {
+                "task": "clearAllListeners",
+                "action": "NA",
+                "appType": "<appType>"
+            }
+        },
+        "context": {
+            "source": "device"
+        }
+    }
+```
+
+### Parameters
+
+| Key                   | Description                                                                           | Required? |
+|-----------------------|---------------------------------------------------------------------------------------|-----------|
+| task                  | "clearAllListeners"- Its a static value and should not be changed for this handler    | Y         |
+| appType               | Corresponding intent is launching on which app                                        | Y         |
+
```

Comment:
I liked the sample response provided in CallMethodHandler wondering if it would work here as well, even if it is just a true/false.

Reply:
added response format for all handlers